src/commands/MurderNistery/MurderNistery.js

```diff
@@ -0,0 +1,58 @@
+const BaseCommand = require('../../utils/structures/BaseCommand');
+
+module.exports = class TestCommand extends BaseCommand {
+  constructor() {
+    super('nistery', 'game', []);
+  }
+
+  async run(client, message, args) {
+    if (args[0] && args[0] === "start") {
+        this.gameStart(client, message.channel);
+        return;
+    }
+
+    if (client.nisteryState) {
+        message.channel.send("There's already a game in progress. You're really eager to kill your friends, aren't you? 🧐");
+        return;
+    }
+
+    await this.gameIntro(client, message);
+  }
+
+  async gameIntro(client, message) {
+    client.nistery = {
+        state: 1,  // change this so we don't use magic numbers
```

Comment:
I suggest something like this:
```
const State= {
   START: 1,
   GAME: 2,
   FINISHED: 3,
   INVALID: 4
};

let state = State.START;
```

You can change the names to other things, just wanted to give the example syntax